plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/AbstractDeltaLakePageSink.java

```diff
@@ -154,10 +154,10 @@ public AbstractDeltaLakePageSink(
             DeltaLakeColumnHandle column = inputColumns.get(inputIndex);
             switch (column.getColumnType()) {
                 case PARTITION_KEY:
-                    int partitionPosition = canonicalToOriginalPartitionPositions.get(column.getName());
+                    int partitionPosition = canonicalToOriginalPartitionPositions.get(column.getBaseColumnName());
```

Comment:
> Rename member variables and getter method in DeltaLakeColumnHandle

This shouldn't be a standalone commit.

if the later change adds "baseName" and "partial name", every use-place of `getName` has to make a decision which one to use. Previously, the name identified the column, so I could e.g. `uniqueIndex` by name. I cannot do that with base names, right?

Reply:
Merged commits.

What do you suggest for the method name?
Should it be `getName` -->  For nested columns, this will return the unqualified name of the last field in the path. Whereas for top-level fields its the root name.